aws/table_ aws_appautoscaling_target.go

```diff
@@ -0,0 +1,153 @@
+			},
+			{
+				Name:        "scalable_dimension",
+				Description: "The scalable dimension associated with the scalable target. This string consists of the service namespace, resource type, and scaling property.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "creation_time",
+				Description: " The Unix timestamp for when the scalable target was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "min_capacity",
+				Description: "The minimum value to scale to in response to a scale-in activity.",
+				Type:        proto.ColumnType_INT,
+			},
+			{
+				Name:        "max_capacity",
+				Description: "The maximum value to scale to in response to a scale-out activity.",
+				Type:        proto.ColumnType_INT,
+			},
+			{
+				Name:        "role_arn",
+				Description: "The ARN of an IAM role that allows Application Auto Scaling to modify the scalable target on your behalf.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "suspended_state",
+				Description: "Specifies whether the scaling activities for a scalable target are in a suspended state.",
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// Standard columns for all tables
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ResourceId"),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listAwsApplicationAutoScalingTargets(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	// TODO put me in helper function
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+	plugin.Logger(ctx).Trace("listAwsApplicationAutoScalingTargets", "AWS_REGION", region)
+
+	name := d.KeyColumnQuals["service_namespace"].GetStringValue()
+
+	// Create Session
+	svc, err := ApplicationAutoScalingService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	// List call
+	err = svc.DescribeScalableTargetsPages(
+		&applicationautoscaling.DescribeScalableTargetsInput{
+			ServiceNamespace: &name,
+		},
+		func(page *applicationautoscaling.DescribeScalableTargetsOutput, isLast bool) bool {
+			for _, scalableTarget := range page.ScalableTargets {
+				d.StreamListItem(ctx, scalableTarget)
+
+			}
+			return !isLast
+		},
+	)
+
+	return nil, err
+}
+
+//// HYDRATE FUNCTIONS
+
+func getAwsApplicationAutoScalingTarget(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("getAwsApplicationAutoScalingTarget")
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+
+	name := d.KeyColumnQuals["service_namespace"].GetStringValue()
+	ID := d.KeyColumnQuals["resource_id"].GetStringValue()
```

Comment:
don't use uppercase when defining any local variables, use lowerCamelCase instead. 
```suggestion
	id := d.KeyColumnQuals["resource_id"].GetStringValue()
```